draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,15 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint SHOULD limit the number of in flight RETIRE_CONNECTION_ID frames
+to bound the necessary state. In order to minimize delay in common situations,
+the limit on the number of in flight RETIRE_CONNECTION_IDs SHOULD be at least
+the active_connection_id_limit. An endpoint MAY choose to treat having too many
+connection IDs in need of retirement as a connection error of type
+CONNECTION_ID_LIMIT_ERROR.
```

Comment:
If we are to suggest that endpoints MAY immediately close the connection (an action that leads to loss of HTTP requests / responses), we should give a specific advice on when it would be safe to do so. The only number we provide in this paragraph is `active_connection_id_limit`, but we agree that it is too small to be used for closing the connection. We should be clear about that.

Reply:
I'm happy to include whatever minimum you suggest.

Reply:
I could live with saying "at least twice the active_connection_id_limit" in the first sentence.